_layouts/post.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+custom_js:
+- prism
+- guides
+---
+
+<div>
+  <div class="container-fluid section-ghostwhite">
+    <div class="col-md-3">
+      <div id="all-guides">
+        <label class="pull-right" >
+          <a href="/guides">&lt; All Guides</a>
+        </label>
+      </div>
+        <div id="toc">
+            {% toc %} {{page.document | tocify_asciidoc}}
+        </div>
+    </div>
+    <div class="col-md-7 post-section-white">
+        <div class="col-md-9 col-md-offset-2 post-title text-center">
+            <img src="{{ page.image }}" alt="post logo">
+            <label>{{ page.title }}</label>
+            <p>
+                {% assign words = content | number_of_words %}
+                {% if words < 360 %}
+                  1 min read.
+                {% else %}
+                  {{ words | divided_by:180 }} mins read.
```

Comment:
I wanted to ask today what formula we should use for this because there are various formulas. They also put into consideration text difficulty and reading speed. E.g Someone who reads 150 words at 130wpm can complete in 1.2 minutes. https://wordstotime.com/